btf/src/i2c_test.cpp

```diff
@@ -0,0 +1,36 @@
+#include <gtest/gtest.h>
```

Comment:
This file is easy enough to understand - can you verbosely comment it and we can make this a great example/template to use when developing tests and it should help with people who haven't done this before